### PR TITLE
refactor time weighted average and add unit tests 

### PR DESCRIPTION
**Pull request by cching95**

### CODE REVIEW

File: src/sdk/python/rtdip_sdk/queries/time_series/_query_builder.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def _raw_query(parameters_dict: dict) -> str:
         "{% if include_bad_data is defined and include_bad_data == false %}"
         "AND Status = 'Good'"
         "{% endif %}"
-        "ORDER BY EventTime"
+        "ORDER BY EventTime, TagName"
```
Comment (GBBBAS) [inline src/sdk/python/rtdip_sdk/queries/time_series/_query_builder.py:79]: should this be Tagname, EventTime?

File: tests/sdk/python/rtdip_sdk/queries/test_raw.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
 DATABRICKS_SQL_CONNECT = 'databricks.sql.connect'
 DATABRICKS_SQL_CONNECT_CURSOR = 'databricks.sql.connect.cursor'
 INTERPOLATION_METHOD = "test/test/test"
-MOCKED_QUERY='SELECT from_utc_timestamp(EventTime, "+0000") as EventTime, TagName, Status, Value FROM `mocked-buiness-unit`.`sensors`.`mocked-asset_mocked-data-security-level_events_mocked-data-type` WHERE EventDate BETWEEN to_date(to_timestamp("2011-01-01T00:00:00+00:00")) AND to_date(to_timestamp("2011-01-02T23:59:59+00:00")) AND EventTime BETWEEN to_timestamp("2011-01-01T00:00:00+00:00") AND to_timestamp("2011-01-02T23:59:59+00:00") AND TagName in (\'MOCKED-TAGNAME\') ORDER BY EventTime'
+MOCKED_QUERY='SELECT from_utc_timestamp(EventTime, "+0000") as EventTime, TagName, Status, Value FROM `mocked-buiness-unit`.`sensors`.`mocked-asset_mocked-data-security-level_events_mocked-data-type` WHERE EventDate BETWEEN to_date(to_timestamp("2011-01-01T00:00:00+00:00")) AND to_date(to_timestamp("2011-01-02T23:59:59+00:00")) AND EventTime BETWEEN to_timestamp("2011-01-01T00:00:00+00:00") AND to_timestamp("2011-01-02T23:59:59+00:00") AND TagName in (\'MOCKED-TAGNAME\') ORDER BY EventTime, TagName'
```
Comment (GBBBAS) [inline tests/sdk/python/rtdip_sdk/queries/test_raw.py:31]: should this be Order BY Tagname, EventTime?

File: src/sdk/python/rtdip_sdk/queries/time_series/_query_builder.py (hunk 1 above)
Comment (cching95) [reply src/sdk/python/rtdip_sdk/queries/time_series/_query_builder.py]: fixed

File: tests/sdk/python/rtdip_sdk/queries/test_raw.py (hunk 2 above)
Comment (cching95) [reply tests/sdk/python/rtdip_sdk/queries/test_raw.py]: fixed

**Review by GBBBAS — APPROVED** [review]

**Pull request merged**